pkg/arvo/app/graph-store.hoon

```diff
@@ -257,11 +257,17 @@
               ?&  !(~(has by archive) resource)
                   !(~(has by graphs) resource)
           ==  ==
+      =/  old-graph=(unit marked-graph:store)
```

Comment:
This is the wrong behavior. We cannot both have an archive of an old graph of a name AND have an active graph of the same name. Overwrite does what it says on the box. It overwrites the data.